src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -216,4 +216,22 @@ public void repoConfig_withoutFormatsAndCliFormats_useDefaultFormats() throws Pa
         Assert.assertEquals(1, actualConfigs.size());
         Assert.assertEquals(ArgsParser.DEFAULT_FORMATS, actualConfigs.get(0).getFormats());
     }
+
+    @Test
+    public void equal() throws InvalidLocationException {
```

Comment:
Please use `three part format` for test methods.
`i.e.` featureUnderTest_testScenario_expectedBehavior()

Reply:
can you suggest a name? cause it tests the equal() method

Reply:
Technically you are using `equals()` for testing and not testing it.

From the assertion, I believe you are testing for `repoConfig_sameLocationDifferentBranch_notSame`?

```
Assert.assertNotEquals(emptyLocationEmptyBranchRepoConfig, validLocationValidBranchRepoConfig);
Assert.assertNotEquals(validLocationDefaultBranchRepoConfig, validLocationValidBranchRepoConfig);
```
However, I am not sure why empty RepoConfigs with empty location are equal since it compares with `Objects.hash(location, branch);`

```
Assert.assertEquals(emptyLocationDefaultBranchRepoConfig, emptyLocationEmptyBranchRepoConfig);
Assert.assertEquals(emptyLocationWithBranchRepoConfig, emptyLocationEmptyBranchRepoConfig);
```

Reply:
Updated the comment above.

Reply:
Thanks for your advice